vendor/bat-native-ads/src/bat/ads/internal/static_values.h

```diff
@@ -68,6 +68,11 @@ const uint64_t kRetryDownloadingCatalogAfterSeconds =
 const std::map<int, std::vector<std::string>> kSupportedRegionsSchemas = {
```

Comment:
global objects with non-trivial destructor are prohibited by the style guide (see examples utilizing base::NoDestrutor in the code)

Reply:
As unrelated to this ticket, I have raised https://github.com/brave/brave-browser/issues/9164, thanks